plugin/parser/ast/rename_column_stmt.go

```diff
@@ -0,0 +1,11 @@
+package ast
+
+// RenameColumnStmt is the rename column statement.
+// For PostgreSQL dialect is the ALTER TABLE RENAME COLUMN.
+type RenameColumnStmt struct {
+	node
+
+	Table   *TableName
```

Comment:
By looking at this, one may wonder why Table is defined as TableName instead of TableDef

Reply:
The `ColumnDef` contains column name, column type, and other members. Here I only add the column name in `ColumnDef` because the other members are useless now. I'll add them if needed in the following PR.